src/map/status.cpp

```diff
@@ -2767,7 +2767,10 @@ int32 status_calc_mob_(struct mob_data* md, uint8 opt)
 			;
 		else
 			md->level = md->db->lv;
+		if (!md->db->title.empty())
+			safestrncpy(md->ud.title, md->db->title.c_str(), NAME_LENGTH);
```

Comment:
```suggestion
		safestrncpy(md->ud.title, md->db->title.c_str(), NAME_LENGTH);
```
It is okay to set it to empty string/copy empty string.